engine/src/mac-qt-player.mm

```diff
@@ -167,6 +227,8 @@ inline NSComparisonResult do_QTTimeCompare (QTTime time, QTTime otherTime)
 {
 	m_movie = [[NSClassFromString(@"QTMovie") movie] retain];
 	m_view = [[NSClassFromString(@"QTMovieView") alloc] initWithFrame: NSZeroRect];
+    m_helper = [[com_runrev_livecode_MCQTKitHelper alloc] init];
```

Comment:
Since you are calling a class method there's no need to create a 'helper' instance.

(Essentially you are just using the MCQTKitHelper class as an easy way to define an obj-c method to patch into the subclass).